### PR TITLE
LB-201: Make column names in statistics tables singular

### PR DESCRIPTION
**Pull request by paramsingh**

### CODE REVIEW

File: admin/sql/create_tables.sql [hunk 1]
```diff
@@ -46,19 +46,19 @@ ALTER TABLE api_compat.session ADD CONSTRAINT session_sid_uniq UNIQUE (sid);
...
 CREATE TABLE statistics.artist (
     id                      SERIAL, -- PK
     msid                    UUID NOT NULL,
     name                    VARCHAR,
-    releases                JSONB,
-    recordings              JSONB,
-    users                   JSONB,
+    release                 JSONB,
+    recording               JSONB,
+    "user"                  JSONB,
```
Comment (alastair) [inline admin/sql/create_tables.sql]: this is a bit annoying - I'm in favour of singular names, but user is a reserved keyword in postgres. Is there some other name we could give to this?

Comment (paramsingh) [reply admin/sql/create_tables.sql]: What do you suggest? `listener`?

Comment (mayhem) [reply admin/sql/create_tables.sql]: listener sounds ok to me.

**Comment by mayhem** [comment]
Merged via command line.

**Pull request closed without merging**